src/ui/views/ConfiguratorView/index.tsx

```diff
@@ -1012,13 +1013,42 @@ const ConfiguratorView: FunctionComponent<ConfiguratorViewProps> = (props) => {
                     <Box>Logs</Box>
                     <Box>
                       <IconButton
-                        aria-label="Copy the logs"
+                        aria-label="Copy log to clipboard"
+                        title="Copy log to clipboard"
                         onClick={async () => {
                           await navigator.clipboard.writeText(logs);
                         }}
                       >
                         <ContentCopy />
                       </IconButton>
+                      <IconButton
+                        aria-label="Save log to file"
+                        title="Save log to file"
+                        onClick={async () => {
```

Comment:
I think this one deserves to be extracted into separate function instead of being inline.